src/test/java/AddSongHmw17.java

```diff
@@ -0,0 +1,51 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class AddSongHmw17 extends BaseTest {
+    By clickViewAllBtn = By.xpath("//button[@data-testid='home-view-all-recently-played-btn']");
+    By clickSongFromList = By.xpath("//section[@id='recentlyPlayedWrapper']//tr[@class='song-item']");
+    By clickAddToBtn = By.xpath("//button[@data-test='add-to-btn']");
+    By addToSuperPlaylist = By.xpath("//section[@id='recentlyPlayedWrapper']//li[contains(text(),'super')]");
+    By clickOnSuperPlaylistLink = By.xpath( "//a[contains(text(),'super')]");
+
+    @Test
+    public void addSongToPlaylist() throws InterruptedException {
+    provideEmail("demo@class.com");
+    providePassword("te$t$tudent");
+    clickSubmitBtn();
+    clickViewAllBtn();
+    String getSongTitle = getSongTitleTxt();
+    Thread.sleep(3000);
+    clickSongFromList();
+    clickAddToBtn();
+    addToSuperPlaylist();
+    clickOnSuperPlaylistLink();
+    String getSongTitleAddedToSuperPlaylist = getSongAddedToSuperPlaylist();
+    Assert.assertEquals(getSongTitleAddedToSuperPlaylist, getSongTitle);
```

Comment:
**Test failed!**

![image](https://user-images.githubusercontent.com/104761482/201328771-5a2d8dbb-7234-4a97-9c11-403d3168ea44.png)

![image](https://user-images.githubusercontent.com/104761482/201329048-207cd593-5d76-4206-a96a-e15477ece9a4.png)

You can try like this:

```
  private String getSongAddedToSuperPlaylist() {
    WebElement songAdded = driver.findElement(By.xpath("//td[contains(text(), 'Pluto')]"));
    return songAdded.getAttribute("innerHTML");
  }
```
![image](https://user-images.githubusercontent.com/104761482/201335717-ce147cfc-d3e5-4074-af63-dc0b20b4f157.png)

How to get HTML code of a WebElement in Selenium? https://www.tutorialspoint.com/how-to-get-html-code-of-a-webelement-in-selenium